providers/src/airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -38,9 +38,14 @@
 from kubernetes.dynamic import DynamicClient
 from sqlalchemy import or_, select, update
 
+try:
+    from airflow.cli.cli_config import ARG_LOGICAL_DATE
+except ImportError:  # 2.x compatibility.
+    from airflow.cli.cli_config import (  # type: ignore[attr-defined, no-redef]
+        ARG_EXECUTION_DATE as ARG_LOGICAL_DATE,
+    )
```

Comment:
We probably could move it to common.compat provider?

Reply:
Part of other PR